app/app.go

```diff
@@ -348,6 +358,8 @@ type App struct {
 	configurator module.Configurator
 
 	consensusMessageAdmissionHandler consensus.AdmissionHandler
+
+	dymintVersionGetter func() (uint32, error)
```

Comment:
not sure I understand why is this necessary?

Reply:
so it can be mocked for testing